tests/integration/components/bs-dropdown/menu/link-to-test.js

```diff
@@ -0,0 +1,32 @@
+import { find, findAll } from 'ember-native-dom-helpers';
+import { moduleForComponent } from 'ember-qunit';
+import { testBS3, testBS4 } from '../../../../helpers/bootstrap-test';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('bs-dropdown/menu/link-to', 'Integration | Component | bs dropdown/menu/link to', {
+  integration: true
+});
+
+testBS3('it has correct markup', function(assert) {
+  // Template block usage:
+  this.render(hbs`
+    {{#bs-dropdown/menu/link-to}}
```

Comment:
Seems tests are failing because of the missing route parameter ("index" should work) for the link component!